playwright/tests/parallel-3/liveblog-ad-limit.spec.ts

```diff
@@ -26,81 +26,73 @@ const addAndAwaitNewBlocks = async (page: Page, blockContent: string) => {
 		// @ts-expect-error -- browser land
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-call -- browser land
 		window.mockLiveUpdate({
-			numNewBlocks: 7,
+			numNewBlocks: 10,
 			html: `
-				<p style="height:1000px;" class="pending block">${blockContent}</p>
-				<p style="height:1000px;" class="pending block">New block</p>
-				<p style="height:1000px;" class="pending block">New block</p>
-				<p style="height:1000px;" class="pending block">New block</p>
-				<p style="height:1000px;" class="pending block">New block</p>
-				<p style="height:1000px;" class="pending block">New block</p>
-				<p style="height:1000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">${blockContent}</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
+				<p style="height:2000px;" class="pending block">New block</p>
 			`,
 			mostRecentBlockId: 'abc',
 		});
 	}, blockContent);
+
 	// click on the toast to insert and scroll to the new blocks
-	await page.getByRole('button', { name: '7 new updates' }).click();
+	await page.getByRole('button', { name: '10 new updates' }).click();
 
 	// Here, we force the test to wait until the blocks are visible on the page before counting the ad slots
 	const newBlock = page.getByText(blockContent).first();
 	await expect(newBlock).toBeVisible();
 };
 
-const countInlineSlots = (page: Page) =>
-	page.locator('#liveblog-body .ad-slot--liveblog-inline').count();
-
-test.describe.serial('Ad slot limits', () => {
+test.describe('A minimum amount of ad slots load', () => {
```

Comment:
There are two main checks in this test:
1. when there is less than the maximum amount of ads and new content blocks are inserted, check that we don't insert ads past the limit
2. when there is currently the maximum amount of ads and new content blocks are inserted, check that we don't insert ads past the limit

which in theory should be two separate tests. However, I found that combining these into one test made the pass rate much more reliable. I didn't receive any "mockLiveUpdate not available errors" when there was one test rather than two